tests/test_simulator.py

```diff
@@ -23,4 +26,5 @@ def test_expand_double_gate():
                 bits[right] = 1
                 state_index = sum([2**i for i, b in enumerate(reversed(bits))
                                    if (b==1)])
-                assert qubits.distribution()[state_index] == 1.0
+                assert np.isclose(qubits.distribution()[state_index],
```

Comment:
Very minor: I think we should either enforce PEP or allow long lines (see two rows above). Mixing long lines with ugly line breaks is the worst of two worlds.

Reply:
agree, I think enforcing pep make sense.